### PR TITLE
Retrieve version from package.json at build time

### PR DESCRIPTION
**Pull request by Andarist**
Less maintenance 💰

### CODE REVIEW

File: src/ReactFinalForm.js [hunk 1]
```diff
@@ -22,8 +22,9 @@ import isSyntheticEvent from './isSyntheticEvent'
 import type { FormRenderProps } from './types.js.flow'
 import ReactFinalFormContext from './context'
 import useLatest from './useLatest'
+import { version } from '../package.json'
 
-export const version = '6.2.0'
+export { version }
```
Comment (erikras) [inline src/ReactFinalForm.js]: Don't suppose you could do `export { version } from '../package.json'`, can you?

**Comment by erikras** [comment]
source

**Comment by erikras** [comment]
This pull request **introduces 2 alerts** when merging 4dcc45aaf9ba4e38a1152f5269662d55bf3ea422 into f8ca858011ec9ae940fcf6ce74d7d82ca868c81f - [view on LGTM.com](https://lgtm.com/projects/g/final-form/react-final-form/rev/pr-39fd5bcdcdd25f7c0e216013c61b6f3faba133b3)

**new alerts:**

* 1 for Assignment to constant
* 1 for Syntax error

---

This project has automated code review enabled, but doesn't use the <a href="https://github.com/apps/lgtm-com">LGTM GitHub App</a>. Migrate over by <a href="https://github.com/apps/lgtm-com/installations/new">installing the app</a>. Read about the <a href="https://lgtm.com/blog/announcing_github_apps_migration">benefits of migrating to GitHub Apps</a> in the blog.

---

*Comment posted by [LGTM.com](https://lgtm.com)*

File: src/ReactFinalForm.js (hunk 1 above)
Comment (Andarist) [reply src/ReactFinalForm.js]: I've started with that 😅 but you are using this variable when constructing `versions` object in this file and such reexport doesn't create a local binding which could be used.

**Pull request merged**